frontend/js/src/about/add-data/AddData.tsx

```diff
@@ -132,6 +132,19 @@ export default function AddData() {
           , the web version of Tidal running in electron with Hi-Fi (High & Max)
           support
         </li>
+        <li>
```

Comment:
Not sure if it is intentional to duplicate the link to TIDAL Hi-Fi (already there at [lines 128-134](https://github.com/metabrainz/listenbrainz-server/blob/ac3a438c01b3de17023eb98b2c3cb5ded6ed4869/frontend/js/src/about/add-data/AddData.tsx#L128-L134)).

I suspect you wanted to replace that item with the new one, in which case the old one still needs to be removed.

Reply:
I must have been asleep when I was doing these additions... thanks for the catch. The link should not be the same - as far as I can tell TIDAL Hi-Fi and this new link (TIDAL + a custom client) are two different options for users to send listens via TIDAL. I have updated the new link now!